ssl/statem/statem_clnt.c

```diff
@@ -1347,7 +1347,14 @@ static int set_client_ciphersuite(SSL *s, const unsigned char *cipherchars)
     if (s->hit && (s->session->cipher_id != c->id)) {
         if (SSL_IS_TLS13(s)) {
             const EVP_MD *md = ssl_md(s->ctx, c->algorithm2);
-
+           /*
+            * This code eliminates UB if in sslapitest.c at the line #4780
+            * clntsess->cipher = NULL;
```

Comment:
Have you actually seen `clntsess->cipher` become NULL as a result of that line in sslapitest.c? I'm struggling to see how it would occur.  An explicit NULL check in sslapitest.c might be appropriate to prevent it.

Reply:
I'd also remove this comment actually. Pointing to specific line numbers in another file is unlikely to remain accurate for any length of time. As soon as someone makes an edit in sslapitest.c above that line number, it is going to become incorrect.